shared/cfg/base.cfg

```diff
@@ -822,7 +822,7 @@ nvdimm_uuid = ""
 
 
 # Set below params to attach TPM device to
-# guest xml
+# guest xml and qemu command line.
```

Comment:
Please remove the dot at the end of the line.

Reply:
Hi @vivianQizhu 
Thanks, have updated, please help me to review, thanks.